### DISCUSSION TITLE
Startup exception

### DISCUSSION THREAD

**Discussion by ygcnxz** [apache/skywalking#12885] [General]
ERROR 2024-12-20 14:50:23.490 localhost-startStop-1 SkyWalkingAgent : Enhance class org.glassfish.jersey.server.ApplicationHandler error.
org.apache.skywalking.apm.agent.core.plugin.PluginException: Can't create InstanceMethodsAroundInterceptor.
 
How to solve it?

**Comment by wu-sheng** [comment]
Maybe you put the agent into classpath? Or, you don't copy the whole agent package?

Please check the agent logs, rather than only app logs.
